docs/sources/tempo/setup/operator/grafana_datasource.md

```diff
@@ -1,25 +1,41 @@
 ---
-title: Grafana Tempo data source
+title: Grafana data source
 description: Use the Tempo Operator to deploy Tempo and use it as a data source with Grafana
 aliases:
- - /docs/tempo/latest/gateway
- - /docs/tempo/operations/gateway
-weight: 15
+ - /docs/tempo/operator/grafana_datasource
+weight: 400
 ---
 
-# Grafana Tempo data source
+# Grafana data source
 
-You can configure the `TempoStack` to send data to Grafana and configure the Tempo data source.  
+You can use Grafana to query and visualize traces of the `TempoStack` instance by configuring a Tempo data source in Grafana.
+
+## Using Grafana Operator
```

Comment:
Headings should use non-gerunds, for example "Start..." instead of "Starting..." So in this case, it would be "use." 
```suggestion
## Use Grafana Operator
```